src/main.cpp

```diff
@@ -929,10 +933,10 @@ bool parse_build_flags(Array<String> args) {
 						}
 
 						if (ok) switch (bf.kind) {
-						case BuildFlag_Help:
+						case BuildFlag_Help: {
```

Comment:
Please don't add blocks around cases when they were not needed in the first place.

Reply:
Addressed in 8c5c45a04c2a525e048538d764cc35f44239230b.